traefik/config/traefik.toml

```diff
@@ -0,0 +1,6 @@
+logLevel = "{{cfg.log.level}}"
+
+{{~#if cfg.web.enable}}
```

Comment:
Thanks! I'm aware of this, but I wouldn't have known how to deal with this line then
```toml
address = "{{cfg.web.host}}:{{cfg.web.port}}"
```
...if I want, for other reasons, to keep `host` and `port` separate. Like, there's no way to put
```toml
[web]
host = "foo"
port = 3000
address = "{{web.host}}:{{web.port}}"
```
in `default.toml`, is there?